src/main/java/core/gui/theme/ThemeManager.java

```diff
@@ -108,24 +108,24 @@ public Object get(String key){
 		return tmp;
 	}
 
-	private ImageIcon getImageIcon(String key){
+	private Icon getImageIcon(String key){
```

Comment:
Maybe a new name should be chosen if it doesn’t necessarily return an `ImageIcon` anymore.